File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int l=0,m=0,r=nums.length-1;
        while(m<=r){
            if(nums[m]==0){
                int temp=nums[m];
                nums[m]=nums[l];
                nums[l]=temp;
                l++;
                m++;
              //  continue;
            }
            else if(nums[m]==1){
                m++;
              //  continue;
            }
            else if(nums[m]==2){
                int temp=nums[m];
                nums[m]=nums[r];
                nums[r]=temp;
                r--;
                //continue;
            }
        }
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        l=m=0
        h=len(nums)-1
        while m<=h:
            if nums[m]==0:
                t=nums[m]
                nums[m]=nums[l]
                nums[l]=t
                l+=1
                m+=1
            elif nums[m]==1:
                m+=1
            else:
                t=nums[h]
                nums[h]=nums[m]
                nums[m]=t
                h-=1
        print(nums)               